lib_comfyui/comfyui/iframe_requests.py

```diff
@@ -11,27 +10,24 @@
 
 class ComfyuiIFrameRequests:
     finished_comfyui_queue = multiprocessing.Queue()
-    workflow_type_ids = {}
-    param_events = {}
-    last_request = None
-    loop = None
     focused_webui_client_id = None
+    server_instance = None
+    sid_map = {}
 
     @staticmethod
     @ipc.run_in_process('comfyui')
-    def send(request_params):
+    def send(request, workflow_type, data={}):
```

Comment:
pycharm tells me data is ran only once, so any update to data will create weird bugs. maybe just to be sure we should put none and default to a new empty dict